src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +285,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             va("%s://", current_game->gamename_string)
```

Comment:
I think this is one of the strings that should be engine-specific, not game-specific.